api/src/controllers/team.js

```diff
@@ -76,9 +76,44 @@ async function addTeam(req, res) {
 }
 
 /**
- * Gets the team member arrray from the database on /team/:team_id/member.
+ * Gets the team document from the database on /team.
+ * @param {*} req request object, containing team email and password in the body as JSON
+ * @param {*} res response object, the found team document
+ * @returns 200: the team was found
+ * @returns 404: team is not found
+ */
+async function loginTeam(req, res) {
+  Team.findOne({ email: req.body.email })
+    .then((team) => {
+      if (team == null) {
+        res.status(400).send('User not found');
+      } else if (team.password != req.body.password) {
+        res.status(403).send('Incorrect password');
+      } else {
+        console.log(team);
+        res.send(JSON.stringify({ team: team }));
+      }
+    })
+    .catch((err) => res.status(400).json('Error: ' + err));
+}
+
+/**
+ * Handles a POST request to add a team on the endpoint /team.
+ * @param {*} req request object -  json object containing at least two fields - teamName and orgName.
+ * @param {*} res response object - updated team object
+ * @returns 201: returns updated team details
+ */
+async function addTeam(req, res) {
```

Comment:
I think your rebase may have went wrong somewhere since the static analyzer said there's two addTeam functions: lines 106-111 and 71-76

Reply:
yep, thanks for picking this up